lib/capdrupal.rb

```diff
@@ -256,6 +274,50 @@
     end
   end
 
+  namespace :security do
+    desc 'Apply Drupal security recommended layer'
```

Comment:
I in favor of not having these file on prod but I'm not even sure if this is `recommended` by the Drupal community (see [here](https://www.drupal.org/node/766404)). We should probably find another name

Reply:
Okay, I understand. I will remove the recommended then and move everything into obfuscating